### DISCUSSION TITLE
161

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#108] [General]
# 潮流周刊第161期 - 极简台灯

封面图来源于周末买了一个小米黑色的台灯，放到桌上太太太好看了，金属质感很不错，特别适合当做屏幕灯使用，看看可以用多久。

https://weekly.tw93.fun/posts/161-%E6%9E%81%E7%AE%80%E5%8F%B0%E7%81%AF

**Comment by markyun** [comment]
如果周刊右侧可以加上目录就更方便了。

**Reply by tw93** [reply]
哈哈哈 甚至可以提交PR

**Comment by ethanz-code** [comment]
订阅一下，喜欢这种内容分享方式😘

**Comment by liusk98** [comment]
好奇问一下博主使用的macbook的支架是什么？

**Reply by tw93** [reply]
之前用倍思的一款，不过质量太挫了，基本上生命就2个月，后面买了pdd的一款，很稳很稳
https://mobile.yangkeduo.com/goods2.html?refer_share_uin=GIR725DZWLTTIJUGJPLCLNRUFI_GEXDA&_oak_share_time=1715416565&share_uin=GIR725DZWLTTIJUGJPLCLNRUFI_GEXDA&page_from=101&refer_share_id=Xw5RoGrAf0uOY9UQt4ytvICanM3tupgj&refer_share_channel=copy_link&_oak_share_snapshot_num=1580&pxq_secret_key=TEJLYMYS4VTJYXXFIKXOOZSZIRSPGJQU7EUZNT7WRQJJKSIHQ62A&goods_id=363858912588
